### PR TITLE
initial pass at a CONTRIBUTING guide

### PR DESCRIPTION
**Pull request by clarkbw**
Here's my first pass at a CONTRIBUTING document for #51

Things I would like to finalize but didn't get to:
* Git commit message style guide
 * We don't seem to really have a style guide for commit messages yet so I didn't want to generate one
* Better way to describe the JS and CSS style guides
 * The .*rc files do this but I'd like to give an overall description

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+From the original demo the devtools.html project has progressed quite a bit.  To learn more about it please read the [devtools.html proposal document](https://docs.google.com/document/d/1_5aerWTN_GVofr6YQVjmJlaGfZ4nv5YKZmdGHewfTpE/edit#heading=h.dw3amfbdp0lh) and take a look at the [devtools.html meta bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1263750) for tracking progress.
+
+### Firefox DevTools Project
+
+While the medium to long term goal of this project is to land within Firefox.  If you're looking to work directly on the DevTools project which ships developer tools for Firefox and Firefox Developer Edition you can find more information on the Mozilla wiki [DevTools / Get Involved](https://wiki.mozilla.org/DevTools/GetInvolved).
```
Comment (jlongster) [inline CONTRIBUTING.md]: The first sentence is not complete, if you drop the "while" it makes sense

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+* [Fork this project](https://github.com/jlongster/debugger.html#fork-destination-box)
+* Create a branch to start your work `git checkout -b feature-branch`
+* Commit your work
+* Create a pull request
+
+### Pull Requests
+
+* Include screenshots and animated GIFs in your pull request whenever possible.
+* List any steps necessary to trigger the feature you've created or bug you are fixing
+* Always run the tests locally before creating your PR
```
Comment (jlongster) [inline CONTRIBUTING.md]: Link to the README about running tests?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+```
+$ npm run test
+```
+
+
+## Styleguides
+
+### JavaScript Styleguide
+
+We use [eslint](http://eslint.org/) to maintain our JavaScript styles.  The [.eslintrc](https://github.com/jlongster/debugger.html/blob/master/.eslintrc) file contains our style definitions, please adhere to those styles when making changes.
```
Comment (jlongster) [inline CONTRIBUTING.md]: You could link [here](https://wiki.mozilla.org/DevTools/CodingStandards#Code_style) and mention that we follow the greater devtools style guide

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+### Firefox DevTools Project
+
+While the medium to long term goal of this project is to land within Firefox.  If you're looking to work directly on the DevTools project which ships developer tools for Firefox and Firefox Developer Edition you can find more information on the Mozilla wiki [DevTools / Get Involved](https://wiki.mozilla.org/DevTools/GetInvolved).
+
+
+## How Can I Contribute?
+
+### Reporting Bugs :bug:
+
+Because of our unstable nature at this point bug fixing is not a top priority for the team.  If you find an issue with the code please do [file an issue](https://github.com/jlongster/debugger.html/issues/new).  We'll do our best to review the issue but a fix may be coming with future changes and so the issue may be closed out to wait until a future fix comes along.
```
Comment (jlongster) [inline CONTRIBUTING.md]: It's not that bug fixing isn't a priority, but fixing any bugs that relate to out-of-scope features. I would reword this to indicate something like that. Maybe even just saying "fixing all the bugs" instead of "bug fixing" sounds better.

**Comment by jlongster** [comment]
Looks great to me! I'll let you merge it in case you want to change anything from my comments. Feel free to merge whenever you want.

**Comment by jasonLaster** [comment]
Great job. Lets also link to this in the README

**Comment by jasonLaster** [comment]
Merging it. Feel free to make changes/improvements going forward :)
